.gitignore

```diff
@@ -1,3 +1,5 @@
 /php/tests/node_modules
 /vendor
 /phpunit.xml
+.idea/
```

Comment:
Please, use [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for personal settings.